go/vt/sqlparser/parse_test.go

```diff
@@ -2781,6 +2787,17 @@ func TestCreateTable(t *testing.T) {
 			"	constraint second_ibfk_1 foreign key (k, j) references simple (a, b) on update cascade\n" +
 			")",
 
+		// check constraint
+		"create table t (\n" +
```

Comment:
You need test sof two additional forms:

1) On the column, where the keyword CONSTRAINT and the constraint name is missing
2) On the column, where the constraint name is missing

Reply:
working on sans-id, but i omitted on-column checks because they get re-organized on string-formatting